modules/ROOT/pages/authentication-authorization/manage-roles.adoc

```diff
@@ -375,7 +375,7 @@ By default it gives access to the home database and to execute privileges for pr
 * `architect` - can do the same as `publisher` as well as create and manage indexes and constraints.
 * `admin` - can do the same as all the above, as well as manage databases, aliases, users, roles, and privileges.
 
-More information about the built-in roles and their privileges can be found in xref:authentication-authorization/built-in-roles[].
+More information about the built-in roles and their privileges can be found in xref:authentication-authorization/built-in-roles.adoc[].
```

Comment:
were this also supposed to get a description in the `[]` or does adding `.adoc` solve that?

Reply:
It seems to work https://neo4j-docs-operations-1047.surge.sh/operations-manual/5/authentication-authorization/manage-roles/